frontend/containers/DocumentEdit/DocumentEdit.jsx

```diff
@@ -143,13 +133,24 @@ class DocumentEdit extends Component {
     this.currentCollection = currentCollection
   }
 
+  handleRoomChange() {
+    const {state, actions, documentId} = this.props
```

Comment:
Could you add [a space after initialisation](https://github.com/dadi/publish/blob/master/docs/coding-guidelines.md#white-space)?